homeassistant/components/nad/media_player.py

```diff
@@ -45,7 +45,7 @@
 CONF_VOLUME_STEP = "volume_step"  # for NADReceiverTCP
 CONF_SOURCE_DICT = "sources"  # for NADReceiver
 
-SOURCE_DICT_SCHEMA = vol.Schema({vol.Range(min=1, max=10): cv.string})
+SOURCE_DICT_SCHEMA = vol.Schema({vol.Range(min=1, max=20): cv.string})
```

Comment:
I suggest that you change it to 12, as matches your configuration with a single expansion card.
You should also probably add a comment to indicate the way the max was chosen.